packages/auto-icons/src/index.ts

```diff
@@ -11,18 +11,19 @@ export default defineWxtModule<AutoIconsOptions>({
   async setup(wxt, options) {
     const parsedOptions = defu(options, {
       enabled: true,
-      baseIconsPath: resolve(wxt.config.srcDir, 'assets/icon.png'),
+      baseIconPath: resolve(wxt.config.srcDir, 'assets/icon.png'),
       grayscaleOnDevelopment: true,
       sizes: [128, 48, 32, 16],
     });
 
+    const resolvedPath = resolve(process.cwd(), parsedOptions.baseIconPath);
```

Comment:
If the option defaults to resolving from the src dir, should we also resolve from there instead of cwd?

Maybe also update the options jsdoc to specify where the path is relative to?

Reply:
Changed it to resolve from srcDir and added a comment in the options.